notebooks/welcome.ipynb

```diff
@@ -22,16 +22,16 @@
     "## Other useful links\n",
     "- [The Landlab Documentation](https://landlab.readthedocs.io/en/latest/)\n",
     "- [The Landlab code base](https://github.com/landlab/landlab)\n",
+    "- [The Landlab user guide](https://landlab.readthedocs.io/en/latest/user_guide/)\n"
```

Comment:
yes!

Reply:
agreed. though a missing comma at the end of this line broke the tests. I've fixed it.